screens/Providers.js

```diff
@@ -54,5 +74,23 @@ const styles = StyleSheet.create({
     container: {
         flex: 1,
         backgroundColor: "#9bcdd5",
+    },
+    addButton: {
+        backgroundColor: 'white',
+        padding: 10,
+        width: 110,
+        marginBottom: 10,
+        borderRadius: 10,
+        left:10
+    },
+    backButton: {
```

Comment:
The backButton for the rest of the screens is on the top left, is there a reason we've moved it to the top right for these screens?

Reply:
I didn't know the back button was functional on the other screens, since its a form it seemed like a good choice to have the back button on the right as you're not leaving the screen, just the form. Feel Free to change.